src/server/page_server.cpp

```diff
@@ -207,78 +221,30 @@ page_server::tran_server_connection_handler::receive_start_catch_up (tran_server
   unpacker.unpack_int (port);
   cublog::lsa_utils::unpack (unpacker, catchup_lsa);
 
-  _er_log_debug (ARG_FILE_LINE, "[CATCH-UP] Start catching up with the follower (%s:%d), until LSA = (%lld|%d)\n",
+  _er_log_debug (ARG_FILE_LINE,
+		 "[CATCH-UP] It's been requested to start catch-up with the follower (%s:%d), until LSA = (%lld|%d)\n",
```

Comment:
Nit.
Use `CATCH_UP` as it is one "word".
```suggestion
		 "[CATCH_UP] It's been requested to start catch-up with the follower (%s:%d), until LSA = (%lld|%d)\n",
```

Reply:
done.